config/targets.json

```diff
@@ -5,7 +5,8 @@
       {
         "url": "https://openopps-staging.18f.gov/"
       }
-    ]
+    ],
+    "slack_channel": "openopps-platform"
```

Comment:
That's the approach I'd take - have volunteers opt in to the notifications. My thought on this is that we'd merge this with all dummy channels. Once we see it working we point folks to those notifications, get feedback on them, and then start switching them over to 'real' channels.